### PR TITLE
Election module/resolution nft

### LINKED ISSUE #722: Election resolution

### LINKED ISSUE #767: Membership in first epoch

### PR DESCRIPTION
**Pull request by ajsantander**
Fixes #722
Fixes #767

### CODE REVIEW

File: packages/core-modules/contracts/submodules/election/ElectionCredentials.sol [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        }
+    }
+
+    function _removeAllCouncilMembers() internal {
+        SetUtil.AddressSet storage members = _electionStore().councilMembers;
+
+        uint numMembers = members.length();
+
+        for (uint memberIndex = 0; memberIndex < numMembers; memberIndex++) {
+            _removeCouncilMember(members.valueAt(1));
```
Comment (mjlescano) [inline packages/core-modules/contracts/submodules/election/ElectionCredentials.sol:40]: I think here's a typo:
```suggestion
            _removeCouncilMember(members.valueAt(memberIndex + 1));
```

Comment (mjlescano) [reply packages/core-modules/contracts/submodules/election/ElectionCredentials.sol:40]: (Not sure if it should be memberIndex or memberIndex + 1)

Comment (ajsantander) [reply packages/core-modules/contracts/submodules/election/ElectionCredentials.sol:40]: It works by always removing the first member until there are no more members.

Comment (leomassazza) [reply packages/core-modules/contracts/submodules/election/ElectionCredentials.sol:40]: could you comment that in the code to easy reading it (and not think there's a typo)

**Review by leomassazza — APPROVED** [review]

**Review by mjlescano — APPROVED** [review]

File: packages/core-modules/contracts/submodules/election/ElectionCredentials.sol (hunk 1 above)
Comment (ajsantander) [reply packages/core-modules/contracts/submodules/election/ElectionCredentials.sol:42]: Done

**Pull request merged**